### PR TITLE
mrc-1530 Prevalence graph endpoints

### PR DESCRIPTION
**Pull request by EmmaLRussell**
This doesn't include a test for hitting the endpoint itself - I think in hint we just rolled that into the integration testing for the front end actions, so we can do the same here when the front end work goes in. 

To test manually, `scripts/run-development-dependencies.sh`, run the app, then
Get config: `curl http://localhost:8080/impact/graph/prevalence/config`
Get data: `curl --header "Content-Type: application/json" --request POST --data '{}' http://localhost:8080/impact/graph/prevalence/data`

### CODE REVIEW

File: src/app/src/main/kotlin/org/imperial/mrc/mint/Extensions.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package org.imperial.mrc.mint
```
Comment (EmmaLRussell) [inline src/app/src/main/kotlin/org/imperial/mrc/mint/Extensions.kt:1]: This functionality copied from HINT

File: src/app/src/test/kotlin/org/imperial/mrc/mint/helpers/JSONValidator.kt [hunk 2]
```diff
@@ -0,0 +1,93 @@
+package org.imperial.mrc.mint.helpers
```
Comment (EmmaLRussell) [inline src/app/src/test/kotlin/org/imperial/mrc/mint/helpers/JSONValidator.kt:1]: This class copied from HINT - one difference is that hintr has a Response schema for the whole response enclosing `data` which the validator checks for. This isn't in mintr yet.

**Comment by EmmaLRussell** [comment]
Oh of course - I'd completely misremembered and thought the integration tests were just testing things the app used, not the app itself. OK, well why don't I add those as part of this ticket so it's fully tested, but I'll do it in another branch to keep the PR size done. 

> We do actually have endpoint integration tests in hint: https://github.com/mrc-ide/hint/tree/master/src/app/src/test/kotlin/org/imperial/mrc/hint/integration

**Comment by hillalex** [comment]
> Oh of course - I'd completely misremembered and thought the integration tests were just testing things the app used, not the app itself. OK, well why don't I add those as part of this ticket so it's fully tested, but I'll do it in another branch to keep the PR size down.
> 
> > We do actually have endpoint integration tests in hint: https://github.com/mrc-ide/hint/tree/master/src/app/src/test/kotlin/org/imperial/mrc/hint/integration

Awesome, sounds good

**Comment by EmmaLRussell** [comment]
OK, PR for endpoint tests is here: https://github.com/mrc-ide/mint/pull/6

**Pull request merged**